### ISSUE TITLE
Improve error message: "Compression is required"

### ISSUE THREAD

**Issue by oesteban** [poldracklab/fmriprep#977]
Right now:

```
RuntimeError: Compression is required, but svgo or cwebp are not installed
```

it is unclear whether one of them or both are required.

**Comment by effigies** [comment]
I thought this was supposed to be dynamic, as in, we use them if available and not specifically disabled. Throwing an error indicates that we're making it a requirement, which I think is probably a bug.

**Comment by effigies** [comment]
Fixed in #1081.
